Description: Add contributors
 The list of contributors is missing from the source tarball. This list is
 generated from upstream repository, running:
 git shortlog -sne v1.1.3 | cut -c8-
 See https://github.com/LMMS/lmms/issues/2914 for more information.
Author: Javier Serrano Polo <javier@jasp.net>

Index: lmms-1.1.3/doc/CONTRIBUTORS
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ lmms-1.1.3/doc/CONTRIBUTORS	2016-07-12 00:41:47.000000000 +0200
@@ -0,0 +1,77 @@
+Tobias Doerffel <tobias.doerffel@gmail.com>
+Vesa <contact.diizy@nbl.fi>
+Javier Serrano Polo <jasp00@terra.es>
+Paul Giblock <drfaygo@gmail.com>
+Tres Finocchiaro <tres.finocchiaro@gmail.com>
+Lukas W <lukaswhl@gmail.com>
+Raine M. Ekman <raine@iki.fi>
+Wong Cho Ching <chwong249664@yahoo.com>
+Hannu Haahti <grejppi@gmail.com>
+Danny McRae <khjklujn@yahoo.com>
+Dave French <dave.french3@googlemail.com>
+Daniel Winzen <d@winzen4.de>
+Andreas Brandmaier <andy@brandmaier.de>
+Andrew Kelley <superjoe30@gmail.com>
+Oskar Wallgren <oskar.wallgren13@gmail.com>
+Mike Choi <rdavidian71@gmail.com>
+Alexandre Almeida <wilsalx@gmail.com>
+NoiseByNorthwest <noisebynorthwest@gmail.com>
+Johannes Lorenz <johannes89@mailueberfall.de>
+Stian Jørgensrud <stianahj@gmail.com>
+falkTX <falktx@gmail.com>
+Csaba Hruska <csaba.hruska@gmail.com>
+StakeoutPunch <ehr@ravemail.com>
+ma2moto <you.are.being.lied.to.kns@gmail.com>
+mikobuntu <chrissy.mc.1@hotmail.co.uk>
+8tab <8tab@wp.pl>
+Matthew Krafczyk <krafczyk.matthew@gmail.com>
+Spekular <spekularr@gmail.com>
+Umcaruje <uros.maravic@gmail.com>
+DeRobyJ <derobyj@gmail.com>
+Jonathan Aquilina <eagles051387@gmail.com>
+ra <symbiants@gmail.com>
+wongcc966422 <wongcc966422@users.noreply.github.com>
+Gurjot Singh <bhattigurjot@gmail.com>
+Janne Sinisalo <janne.m.sinisalo@gmail.com>
+Krzysztof Foltman <wdev@foltman.com>
+Lou Herard <lherard@gmail.com>
+Paul Wayper <paulway@mabula.net>
+Rüdiger Ranft <rudi@qzzq.de>
+Yann Collette <ycollette.nospam@free.fr>
+grindhold <grindhold@gmx.net>
+midi-pascal <midi-pascal@videotron.ca>
+unfa <unfa00@gmail.com>
+Ian Sannar <ian.sannar@gmail.com>
+Jaroslav Petrnoušek <baegus@gmail.com>
+LYF610400210 <lyfjxymf@sina.com>
+Rafael Ruggiero <rafael46w@gmail.com>
+psyomn <lethaljellybean@gmail.com>
+quadro <quadro2@email.cz>
+sarahkeefe <sarahjanekeefe@gmail.com>
+Achim Settelmeier <lmms@m1.sirlab.de>
+André Hentschel <nerv@dawncrow.de>
+Armin Kazmi <armin.kazmi@tu-dortmund.de>
+Attila Herman <attila589@gmail.com>
+Christopher A. Oliver <oliver@onion.private>
+Devin Venable <venable.devin@gmail.com>
+Fastigium <fastigiummusic@gmail.com>
+Frank Mather <tinycat2001@gmail.com>
+Frederik <freggy@gmail.com>
+Hexasoft <yperret@in2p3.fr>
+Jens Lang <jenslang@users.sf.net>
+Jesse Dubay <jesse@thefortytwo.net>
+Joel Muzzerall <joel.muzzerall@gmail.com>
+Kristi <kristi.dev@gmail.com>
+Markus Elfring <elfring@users.sourceforge.net>
+Nikos Chantziaras <realnc@arcor.de>
+Paul Nasca <zynaddsubfx_AT_yahoo com>
+Peter Nelson <peter1138@users.sourceforge.net>
+Ra <symbiants@gmail.com>
+Steffen Baranowsky <BaraMGB@freenet.de>
+Thorsten Müller <thorsten.muller.ger@googlemail.com>
+TonyChyi <tonychee1989@gmail.com>
+devin <devin@devin-studio.(none)>
+dnl-music <baza-nsk@ya.ru>
+fundamental <mark.d.mccurry@gmail.com>
+groboclown <matt@groboclown.net>
+zm1990s <zm1990s@gmail.com>
